op-node/sources/caching/cache.go

```diff
@@ -0,0 +1,43 @@
+package caching
+
+import lru "github.com/hashicorp/golang-lru"
+
+type Metrics interface {
+	CacheAdd(label string, cacheSize int, evicted bool)
+	CacheGet(label string, hit bool)
+}
+
+// LRUCache wraps hashicorp *lru.Cache and tracks cache metrics
+type LRUCache struct {
+	m     Metrics
+	label string
+	inner *lru.Cache
+}
+
+func (c *LRUCache) Get(key interface{}) (value interface{}, ok bool) {
```

Comment:
Curious what you think about using `any` vs `interface{}` here

Reply:
I just copied the interface that the underlying cache uses. I'm fine with `any` too.